code/controllers/subsystem/jukeboxes.dm

```diff
@@ -13,6 +13,16 @@
 #define JUKE_FALLOFF 4
 #define JUKE_SOUND 5
 
+// Track data
+/// Name of the track
+#define TRACK_NAME 1
+/// Length of the track (in deciseconds)
+#define TRACK_LENGTH 2
+/// BPM of the track (in deciseconds)
+#define TRACK_BEAT 3
+/// Unique identifier for this track (IT DOES NOTHING AND LACKING THIS WILL BREAK THE ENTIRE THING)
```

Comment:
The identifier is used for portable boomboxes, which have a whitelist of specific songs they can play. There's a donor item that has a song list restricted to specific songs. There were plans long ago to add songs that can be played only if the jukebox is emagged, which would've made use of TRACK_ID, but those plans never came to fruition

```suggestion
/// Unique code-facing identifier for this track
```